Dockerfile.ppc64le.ubi

```diff
@@ -0,0 +1,275 @@
+        -DARROW_JEMALLOC=ON \
+        -DARROW_BUILD_STATIC="OFF" \
+        -DARROW_PARQUET=ON \
+        .. && \
+    make install -j ${MAX_JOBS:-$(nproc)} && \
+    cd ../../python/ && \
+    uv pip install -v -r requirements-wheel-build.txt && \
+    PYARROW_PARALLEL=${PYARROW_PARALLEL:-$(nproc)} \
+    python setup.py build_ext \
+    --build-type=release --bundle-arrow-cpp \
+    bdist_wheel --dist-dir /arrowwheels/
+
+###############################################################
+# Stage to build opencv
+###############################################################
+FROM base-builder AS cv-builder
+
+ARG MAX_JOBS
+ARG OPENCV_VERSION=84
+ARG ENABLE_HEADLESS=1
+RUN --mount=type=cache,target=/root/.cache/uv \
+    source /opt/rh/gcc-toolset-13/enable && \
+    git clone --recursive https://github.com/opencv/opencv-python.git -b ${OPENCV_VERSION} && \
+    cd opencv-python && \
+    sed -i 's/"setuptools==59.2.0",/"setuptools<70.0",/g' pyproject.toml && \
+    python -m build --wheel --installer=uv --outdir /opencvwheels/
+
+# build opencv dependent packages
+RUN --mount=type=cache,target=/root/.cache/uv \
+    --mount=type=bind,src=requirements-common.txt,dst=/requirements-common.txt,ro \
+    source /opt/rh/gcc-toolset-13/enable && \
+    uv pip install /opencvwheels/*.whl && \
+    uv pip list | grep opencv && \
+    grep "opencv" requirements-common.txt | uv pip install -r /dev/stdin --no-build-isolation && \
+    uv pip list
+
+###############################################################
+# Stage to build vllm - this stage builds and installs
+# vllm, tensorizer and vllm-tgis-adapter and builds uv cache
+# for transitive dependencies - eg. grpcio
+###############################################################
+FROM base-builder AS vllm-builder
+
+COPY --from=torch-builder /tmp/control /dev/null
+COPY --from=arrow-builder /tmp/control /dev/null
+COPY --from=cv-builder /tmp/control /dev/null
+
+RUN --mount=type=cache,target=/root/.cache/uv \
+    --mount=type=bind,from=torch-builder,source=/torchwheels/,target=/torchwheels/,ro \
+    --mount=type=bind,from=arrow-builder,source=/arrowwheels/,target=/arrowwheels/,ro \
+    --mount=type=bind,from=cv-builder,source=/opencvwheels/,target=/opencvwheels/,ro \
+    --mount=type=bind,src=.,dst=/src/,rw \
+    source /opt/rh/gcc-toolset-13/enable && \
+    uv pip install /opencvwheels/*.whl /arrowwheels/*.whl /torchwheels/*.whl && \
+    sed -i -e 's/.*torch.*//g' /src/pyproject.toml /src/requirements-*.txt && \
+    uv pip install pandas && \
+    uv pip install -r /src/requirements-common.txt -r /src/requirements-cpu.txt -r /src/requirements-build.txt --no-build-isolation && \
+    dnf install -y kmod && \
+    export VLLM_TARGET_DEVICE=cpu && \
+    export GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1 && \
+    cd /src/ && \
+    uv pip install ".[tensorizer]" --no-build-isolation && \
+    uv pip install vllm-tgis-adapter==0.6.3 && \
+    uv build --wheel --out-dir /vllmwheel/ --no-build-isolation
```

Comment:
A couple questions:

- Do we really need to run `pip install` and then re-run `uv build`?
- We're installing `vllm-tgis-adapter` here, is this really required to do that here? Is this done to populate the `uv` cache with all of the `vllm-tgis-adapter` dependencies? If so, it might be worth to add a comment bellow.

Might be also worth it to add a build argument to specify the adapter version since it's hardcoded here and in the final stage.